.circleci/config.yml

```
@@ -89,6 +130,34 @@ jobs:
           file: ./auth0_flutter/example/build/auth0_flutter/reports/jacoco/debug/jacoco.xml
           upload_name: Auth0 Flutter Android
           flags: auth0_flutter_android
+  smoke_test_android:
+    parameters:
+        android_simulator:
+          type: string
+    executor:
+      name: android/android-machine
+      resource-class: xlarge
+      tag: 2021.10.1
```

```suggestion
      resource-class: xlarge
```
This is not necessary.